libs/utils/src/ballot_package.ts

```diff
@@ -200,7 +200,7 @@ async function readBallotPackageFromFilePointer(
 ): Promise<BallotPackage> {
   assert(window.kiosk);
   const zipFile = await openZip(await window.kiosk.readFile(file.path));
-  return readBallotPackageFromZip(zipFile, file.name, file.size);
+  return await readBallotPackageFromZip(zipFile, file.name, file.size);
```

Comment:
This should be equivalent, but doesn't hurt to leave it. Was there an issue here?

Reply:
oh I think I just changed it while debugging something and didn't realize it wasn't there before when switching back to the original code